src/client/steemAPI.js

```diff
@@ -1,9 +1,10 @@
-import { createClient } from 'lightrpc';
+const { Client } = require('busyjs');
+
+const client = new Client('wss://steemd.privex.io');
```

Comment:
We have env variables for this, but they are used for HTTPS endpoint. How do we handle this now, so there are no problems when deploying?

Reply:
We can create a config var like `API_URL` with `wss://steemd.privex.io` as default